bash_completions.go

```diff
@@ -29,14 +30,6 @@ __debug()
     fi
 }
 
-# Homebrew on Macs have version 1.3 of bash-completion which doesn't include
-# _init_completion. This is a very minimal version of that function.
-__my_init_completion()
-{
-    COMPREPLY=()
-    _get_comp_words_by_ref cur prev words cword
-}
```

Comment:
Have to check what _init_completion does in addition.

In any case, we have to call `_get_comp_words_by_ref` with `-n =` to exclude `=` as a word separator. Otherwise, bash 4+ and base <4 will behave differently.